packages/lesswrong/lib/components.ts

```diff
@@ -444,6 +444,7 @@ importComponent("DialogueEditorGuidelines", () => require('../components/posts/d
 importComponent("DialogueEditorFeedback", () => require('../components/posts/dialogues/DialogueEditorFeedback'));
 importComponent("NewDialogueDialog", () => require('../components/posts/NewDialogueDialog'));
 importComponent("DialogueSubmit", () => require('../components/posts/dialogues/DialogueSubmit'));
+importComponent("PostsListSettings", () => require('../components/posts/PostsListSettings'));
```

Comment:
Was this previously not getting imported into the component map?

Reply:
No it did. I'm not quite sure what happened here and why it's being tracked as an addition.